packages/flutter/lib/src/material/slider.dart

```diff
@@ -582,33 +621,55 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
       ),
     );
   }
+  final LayerLink _layerLink = LayerLink();
+
+  OverlayEntry overlayEntry;
+
+  void showValueIndicator() {
+    overlayEntry = OverlayEntry(
```

Comment:
Here is where we should only add the overlay it one has not been added yet (by checking if overlayEntry is null, and assigning it to null when it is removed) 

For both slider and range slider